app/views/devise/registrations/edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<%- content_for :main_class, '--small-centered-content' %>
+<%- content_for :main_class %>
```

Comment:
This page needs to be wider, as it's getting lots more stuff on it than it used to have.

Reply:
If you delete the content-for it should default to the wide layout.